### PR TITLE
Remove powermock from kvm hypervisor plugin

### PR DESCRIPTION
**Pull request by vishesh92**
### Description

This PR...

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by vishesh92** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]

File: plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtVifDriverTest.java [hunk 1]
```diff
@@ -68,8 +64,8 @@ public class LibvirtVifDriverTest {
             "Inactive:         949392 kB\n";
     @Before
     public void setUp() throws Exception {
-        Scanner scanner = new Scanner(memInfo);
-        PowerMockito.whenNew(Scanner.class).withAnyArguments().thenReturn(scanner);
+//        Scanner scanner = new Scanner(memInfo);
+//        PowerMockito.whenNew(Scanner.class).withAnyArguments().thenReturn(scanner);
```
Comment (DaanHoogland) [inline plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtVifDriverTest.java:68]: should this stay in?

File: plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/storage/ScaleIOStoragePoolTest.java [hunk 2]
```diff
@@ -146,35 +149,38 @@ public void testGetPhysicalDiskWithWildcardFileFilter() throws Exception {
...
+            when(ScaleIOUtil.getSystemIdForVolume(volumeId)).thenReturn(systemId);
 
-        // TODO: Mock file exists
-        File file = PowerMockito.mock(File.class);
-        PowerMockito.whenNew(File.class).withAnyArguments().thenReturn(file);
-        PowerMockito.when(file.exists()).thenReturn(true);
+            // TODO: Mock file exists
+//            File file = PowerMockito.mock(File.class);
+//            PowerMockito.whenNew(File.class).withAnyArguments().thenReturn(file);
+//            PowerMockito.when(file.exists()).thenReturn(true);
```
Comment (DaanHoogland) [inline plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/storage/ScaleIOStoragePoolTest.java:179]: should this go?

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by weizhouapache — APPROVED** [review]
code lgtm

only src/test is changed, Trillian test is not required.

**Pull request merged**